array/DNAm/preprocessing/checkRconfigFile.r

```diff
@@ -0,0 +1,164 @@
+##---------------------------------------------------------------------#
+##
+## Title: Check config.r 
+##
+## Purpose of script: Check config file parameters are present and in correct format
+##                    and if not exit the script with error message
+##
+##---------------------------------------------------------------------#
+
+
+#----------------------------------------------------------------------#
+# NOTES
+#----------------------------------------------------------------------#
+
+# Project Folder is provided on the command line
+
+
+#----------------------------------------------------------------------#
+# DEFINE PARAMETERS
+#----------------------------------------------------------------------#
+
+print("checking config.r file parameters are present and correctly formatted...")
+
+'%ni%' <- Negate('%in%') # define '%ni%' (not in)

```

Comment:
This is never used in the file?

Reply:
good point - leftover code and corrected in #209